src/WebJobs.Extensions.ApiHub/Table/ApiHubTableAttribute.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+
+namespace Microsoft.Azure.WebJobs
+{
+    /// <summary>
+    /// Attribute that specifies an ApiHub table or entity.
+    /// </summary>
+    [AttributeUsage(AttributeTargets.Parameter)]
+    public sealed class ApiHubTableAttribute : Attribute
+    {
+        /// <summary>
+        /// Gets the key of the configuration setting that specifies the connection string.
+        /// </summary>
+        public string Key { get; }
+
+        /// <summary>
+        /// Gets the data set name.
+        /// </summary>
+        public string DataSetName { get; set; }
+
+        /// <summary>
+        /// Gets the table name.
+        /// </summary>
+        public string TableName { get; set; }
+
+        /// <summary>
+        /// Gets the entity identifier.
+        /// </summary>
+        public string EntityId { get; set; }
+
+        /// <summary>
+        /// Creates a new instance of this class.
+        /// </summary>
+        /// <param name="key">The key of the configuration setting that 
+        /// specifies the connection string.</param>
+        public ApiHubTableAttribute(string key)
```

Comment:
I don't think this should be a required param - instead, if not specified on attribute, default it to the connection specified on the config object.